policyengine_us/parameters/gov/states/oh/tax/income/rates.yaml

```
@@ -10,21 +10,39 @@ brackets:
       # is below $25,000.
       2021-01-01: 0.01385
   - threshold: 
-      2021-01-01: 25_000
+      values:
+        2021-01-01: 25_000
+        2022-01-01: 26_050
```

As the legal code specifies in 2023 before indexing
```suggestion
        2022-01-01: 26_050
        2023-01-01: 26_050
```